lib/xcvbnm.dart

```diff
@@ -1,33 +1,14 @@
 library xcvbnm;
 
-class Result {
-  // bits
-  double entropy;
-  // estimation of actual crack time, in seconds.
-  double crackTime;
-  // same crack time, as a friendlier string:
-  // "instant", "6 minutes", "centuries", etc.
-  String crackTimeDisplay;
-  // [0,1,2,3,4] if crack time is less than
-  // [10**2, 10**4, 10**6, 10**8, Infinity].
-  // (useful for implementing a strength bar.)
-  int score;
-  // how long it took xcvbnm to calculate an answer,
-  // in milliseconds.
-  int calcTime;
-}
-
-class Xcvbnm {
-  int getNaiveScore(String password) {
-    if (password.length < 3) return 0;
-    if (password.length < 5) return 1;
-    if (password.length < 7) return 2;
-    if (password.length < 9) return 3;
-    if (password.length < 11) return 3;
-    return 4;
-  }
+import 'src/xcvbnm_common.dart' hide Xcvbnm;
+export 'src/xcvbnm_common.dart' hide Xcvbnm;
+export 'src/xcvbnm_impl.dart';
+import 'src/xcvbnm_impl.dart';
```

Comment:
common and impl are seperate libs that can be imported separatly
the real implementation import (and export to get the Xcvbnm, Result and Match definition)
* 'src/xcvbnm_common.dart'
* 'src/xcvbnm_impl.dart'

while the mock implementation imports
* 'src/xcvbnm_common.dart'
* 'src/mock_xcvbnm_impl.dart'